src/main/java/frc/robot/commands/GatherTheCube.java

```diff
@@ -4,21 +4,31 @@
 
 package frc.robot.commands;
 
+import frc.robot.subsystems.EndEffector;
+import frc.robot.subsystems.Arm;
+import frc.robot.Constants;
+
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
 public class GatherTheCube extends CommandBase {
+  EndEffector cubeHarvester = new EndEffector();
+  private Arm robotArm = new Arm();
+  private int scoreMode = EndEffector.m_ScoreMode;
   /** Creates a new GatherTheCube. */
-  public GatherTheCube() {
+  private GatherTheCube() {
     // Use addRequirements() here to declare subsystem dependencies.
   }
-
+  
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {}
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
-  public void execute() {}
+  public void execute() {
+    robotArm.setNewFF();
+    if (scoreMode == 0) robotArm.setAngle(Constants.kPivotMotorGatherAngle);
```

Comment:
when is scoreMode updated? or are you trying to depend on aliasing for the value to change along with m_ScoreMode in EndEffector?

Reply:
Do you need to resend the target angle or is once sufficient?